Pytest infrastructure seems to be broken

### Describe the bug

Recently, the tests that are run on top of the `aiida-core:latest` image started to fail with the following error:

```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

../conda/lib/python3.7/site-packages/aiida/engine/launch.py:77: in run_get_pk

    runner = manager.get_manager().get_runner()

../conda/lib/python3.7/site-packages/aiida/manage/manager.py:251: in get_runner

    self._runner = self.create_runner()

../conda/lib/python3.7/site-packages/aiida/manage/manager.py:285: in create_runner

    settings['communicator'] = self.get_communicator()

../conda/lib/python3.7/site-packages/aiida/manage/manager.py:163: in get_communicator

    self._communicator = self.create_communicator()

../conda/lib/python3.7/site-packages/aiida/manage/manager.py:185: in create_communicator

    url = profile.get_rmq_url()

../conda/lib/python3.7/site-packages/aiida/manage/configuration/profile.py:344: in get_rmq_url

    protocol=self.broker_protocol,

_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <aiida.manage.configuration.profile.Profile object at 0x7fbfaa172150>

    @property

    def broker_protocol(self):

>       return self._attributes[self.KEY_BROKER_PROTOCOL]

E       KeyError: 'broker_protocol'
```

### Steps to reproduce

1. Install latest aiida-core
2. Run test calculation that rely on aiida-pytest infrastructure.

### Examples of errors.

aiida-cp2k: https://travis-ci.org/github/aiidateam/aiida-cp2k/builds/725311241
aiida-raspa: https://travis-ci.org/github/yakutovicha/aiida-raspa

--- a/aiida/manage/tests/__init__.py
+++ b/aiida/manage/tests/__init__.py
@@ -272,6 +272,12 @@ class TemporaryProfileManager(ProfileManager):
             'database_name': self.profile_info.get('database_name'),
             'database_username': self.profile_info.get('database_username'),
             'database_password': self.profile_info.get('database_password'),
+            'broker_protocol': self.profile_info.get('broker_protocol'),
+            'broker_username': self.profile_info.get('broker_username'),
+            'broker_password': self.profile_info.get('broker_password'),
+            'broker_host': self.profile_info.get('broker_host'),
+            'broker_port': self.profile_info.get('broker_port'),
+            'broker_virtual_host': self.profile_info.get('broker_virtual_host'),
             'repository_uri': 'file://' + self.repo,
         }
         return dictionary

